### PR TITLE
Create equipment presets

### PR DESCRIPTION
**Pull request by Supinic**
This would definitely improve long-term usability, as users would no longer be forced to click around and add items one-by-one.

### CODE REVIEW

File: components/AttackerPresets.js [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+		: null;
+
+	const removeAllCustomPresetsElement = (renderedCustomPresets.size > 0)
+		? <><br/><button onClick={clearCustomPresets}>Remove all custom presets</button></>
+		: null;
+
+	return (
+		<div className="highlight-section flex-container-vertical">
+			<h3>Apply setup preset</h3>
+			{...elements}
```
Comment (GaZaTu) [inline components/AttackerPresets.js:240]: i dislike the way you split the UI into variables. (personal preference ofc)
but i would either turn these into their own actual mini components OR just inline them to give the html a visible structure

File: components/AttackerTabs.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ import { AttackerPrayers } from "./AttackerPrayers.js";
 import { AttackerSpells } from "./AttackerSpells.js";
 import { AttackerImport } from "./AttackerImport.js";
 import { AttackerRelics } from "./AttackerRelics.js";
+import AttackerPresets from "./AttackerPresets.js";
 // import Image from "next/image";
 
 class Tab extends Component {
```
Comment (GaZaTu) [inline components/AttackerTabs.js:14]: using class components sucks because hooks are simply superior but also personal preference i guess

**Review by GaZaTu — APPROVED** [review]

File: components/AttackerTabs.js (hunk 2 above)
Comment (Supinic) [reply components/AttackerTabs.js:14]: This one component was refactored from class to function, so that's a bonus

**Comment by Supinic** [comment]
Closed because GitHub doesn't seem to include the 2-3 commits I created after marking this pull request as ready for review(?)

**Pull request closed without merging**